docs/modules/usage/llms/llms.md

```diff
@@ -13,6 +13,11 @@ The following are verified by the community to work with OpenHands:
 * llama-3.1-405b / hermes-3-llama-3.1-405b
 * wizardlm-2-8x22b
 
+:::warning
+OpenHands will issue many prompts to the LLM you configure. Most of these LLMs cost money, so be sure to set spending
+limits and monitor usage.
```

Comment:
Good question. These docs are more for the average user rather than someone in dev mode, so config.toml won't help them--we probably need a deeper "configuration" doc to not confuse people